ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -52,9 +52,9 @@ else
             "root@${IP}" \
             openshift-tests \
             run \
-            "openshift/conformance/parallel" \
+            "openshift/conformance/serial" \
             --dry-run \
-            \| grep -Ff /tmp/test-list \|openshift-tests run -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit -f -
+            \|openshift-tests run -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit -f -
```

Comment:
nit: this could be simplified, you can directly run the suite without a preliminary dry-run, ie `... run -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit "openshift/conformance/parallel"`